Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.h

```diff
@@ -42,6 +42,36 @@ class WebGLFramebuffer;
 class WebGLRenderingContextBase;
 struct XRWebGLLayerInit;
 
+struct WebXRExternalRenderbuffer {
+    GCGLOwnedRenderbuffer rbo;
```

Comment:
Do we normally use this abbreviation? We tend to stay away from all but the most obvious abbreviations in WebKit

Reply:
Oh good observation, I will rename `renderBufferObject`